src/main/java/nl/tudelft/scrumbledore/userinterface/GameDisplay.java

```diff
@@ -98,8 +99,8 @@ public static void launchMultiPlayerGame(Stage passedStage) {
    */
   public static void launchSinglePlayerGame(Stage passedStage) {
```

Comment:
Now that you've got the abstract factory, you don't need two different methods for launching a game anymore. What about creating a method launchGame, that takes a game factory? You can then pass a MultiPlayerGameFactory of a SinglePlayerGameFactory to the method, which it will treat uniformly.
<organisation